types/constants.go

```diff
@@ -0,0 +1,8 @@
+package types
+
+const (
+	// GuardianSetExpiry defines how long a guardian set should remain active for
+	// after being replaced, before then expiring. Currently, 24 hours.
+	GuardianSetExpiry = 86400
+	AddressLength     = 20
+)
```

Comment:
I'm always hesitant to add new source code files for a few lines of code, especially when creating generic files like `constants`, `utils`, `helpers`, etc.

Can these values be moved into the same package/source code file where they are referenced?